sass/core/_parser.scss

```diff
@@ -105,6 +105,44 @@
   @return $needle;
 }
 
+// Next Reference
+// -----
+/// Returns the next available key for parsing/replacing from the given string.
+///
+/// @access private
+///
+/// @param {string} $string -
+///   A string to extract an #alias from
+/// @return {string | null} -
+///   The extracted #alias key to replace, or null if no references exist
+@function _a_next_reference(
```

Comment:
I think we still need tests for `_a_next_reference()`?

Reply:
...I completely forgot I wrote this code. Unit tests coming right up!

Reply:
Unit tests for `_a_next_reference()` added. :)